### ISSUE TITLE
Triplet and tuplet support

### ISSUE THREAD

**Issue by danboid** [stargateaudio/stargate#11]
Stargate currently doesn't have any support for drawing notes 1/3, 1/6 or 1/12 of a beat long, 1/7th etc.

**Comment by stargateaudio** [comment]
I want to make sure I understand correctly...  Stargate DAW does have support for 1/12 notes already by selecting 1/12 from the `Snap:` combobox.  1/3, 1/6, 1/7 are not in there, but should be able to add them easily enough.  Or am I misunderstanding?

**Comment by danboid** [comment]
Yes, snap values which should (optionally) also apply to drawing notes with a CTRL click or something. 1/7th, 1/14th and 1/28th should all be included too if we're going with a bit of 7 time :)

**Comment by stargateaudio** [comment]
OK, should be able to get this done soon.  Help me understand what the subdivision of a beat is on a 1/14 note, Google does not even know what that is :-P

**Comment by stargateaudio** [comment]
Or is the meaning of a "beat" kind of fuzzy in 1/14th?  Some examples of music written this way would great, LOL

**Comment by danboid** [comment]
1/14 is half the duration of a 7, 1/28 is a quarter of as 7th.

I wouldn't worry about 7th and tuplets too much yet, just make sure 1/3, 1/6, 1/12 (and maybe 1/24) notes are all in place because triplets are used in all styles all the time, thanks.
